app/models/episode_story_handler.rb

```diff
@@ -93,12 +94,15 @@ def update_audio
   end
 
   def update_image
-    image_url = begin
-      story.objects['prx:image'].links['original'].href
-    rescue StandardError
-      nil
+    cms_image = story.objects['prx:image'] rescue nil
```

Comment:
Note: I sort of ignored Rubocop in these CMS-handler-y files.  Instead I stuck to existing themes and went for minimal diff.

These will all go away with CMS anyways!